src/sentry/db/models/base.py

```diff
@@ -65,17 +65,41 @@ def set_cached_field_value(self, field_name, value):
         if django.VERSION[:2] < (2, 0):
             setattr(self, f"_{field_name}_cache", value)
             return
+
         self._meta.get_field(field_name).set_cached_value(self, value)
 
+    def get_cached_field_value(self, field_name):
```

Comment:
What do we need this for? Accessing the cached field itself will fetch this value

Reply:
Ahh found your comment in the getsentry pr, missed it earlier since it used a different name